### PR TITLE
Condense utility functions to one file and match ROS twist/wrench order

### PR DESCRIPTION
**Pull request by AdamPettinger**
1. Moves `screw_conversions` implementation to `affordance_utils`
2. Reduces use of inlining
3. Switches order (lin, ang vs ang, lin) of wrenches and twists. This matches ROS and tf2 conversions, but is opposite of what is given in AP papers

### CODE REVIEW

**Review by EAkita — APPROVED** [review]
Looks good!

File: affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp [hunk 1]
```diff
@@ -34,10 +34,10 @@
 #pragma once
 
 #include <optional>
+#include <sstream>
 
 #include <affordance_primitives/msg_types.hpp>
```
Comment (FabianEP11) [inline affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp:39]: maybe I'm missing something but... where do you have this header file?

**Review by FabianEP11 — COMMENTED** [review]
OK.

File: affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp (hunk 1 above)
Comment (AdamPettinger) [reply affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp:39]: This is a C++ std library include, [see here](https://www.cplusplus.com/reference/sstream/stringstream/stringstream/)

Comment (FabianEP11) [reply affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp:39]: Upps sorry wrong line, I didn't mean the stream, I was asking for the <affordance_primitives/msg_types.hpp>

Comment (AdamPettinger) [reply affordance_primitives/include/affordance_primitives/screw_model/affordance_utils.hpp:39]: Ah, that would be in [affordance_primitives/include/affordance_primitives/msg_types.hpp](https://github.com/UTNuclearRobotics/affordance_primitives/blob/noetic/affordance_primitives/include/affordance_primitives/msg_types.hpp)

**Pull request merged**